Core/Core/API/APIUser.swift

```diff
@@ -66,6 +66,15 @@ public struct APIProfile: Codable, Equatable {
     public let name: String
     public let primary_email: String?
     public let login_id: String?
-    public let avatar_url: URL?
+    public let avatar_url: APIURL?
```

Comment:
should we also do the `APIProfile` & `APIUserDisplay` since they have the user's avatar_url as well?